src/relay/backend/build_module.cc

```diff
@@ -438,13 +442,19 @@ class RelayBuildModule : public runtime::ModuleNode {
 
     auto lowered_funcs = graph_codegen_->GetLoweredFunc();
     if (lowered_funcs.size() == 0) {
-      LOG(WARNING) << "no lowered funcs exist in the compiled module";
-    } else {
-      ret_.mod = tvm::build(
-        lowered_funcs,
-        target_host_,
-        BuildConfig::Current());
+      LOG(WARNING) << "No lowered funcs exist in the compiled module, "
+                   << "a dummy function \"__dummy__\" will be created.";
+      Stmt body = EvaluateNode::make(0);
+      Array<ObjectRef> api_args;
+      auto dummy_func = MakeAPI(body, "__dummy__", api_args, 0, false);
+      lowered_funcs.Set("llvm", Array<LoweredFunc>({dummy_func}));
```

Comment:
Is defaulting the LLVM the correct behaviour here (eg. will this fall over if we build without LLVM support)?

Reply:
I think should set `target_host_`. Even we have LLVM support, it is not correct too, imagine our target host is ARM.

Reply:
Thank you guys for your kind comments. We don't need to set target in the latest commit.